host/sgx/create.c

```diff
@@ -639,6 +640,9 @@ oe_result_t oe_create_enclave(
         (flags & OE_ENCLAVE_FLAG_RESERVED) || config || config_size > 0)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+    /* Install the internal ocall function table. */
+    OE_CHECK(oe_register_internal_ocall_function_table());
```

Comment:
Is it better to register the function tables in `_initialize_enclave_host`?

Reply:
Good idea. Resolved.